sdk/containerregistry/container-registry/review/container-registry.api.md

```diff
@@ -57,6 +57,27 @@ export interface ArtifactTagProperties {
     readonly repositoryName: string;
 }
 
+// @public
+export class ContainerRegistryBlobClient {
+    constructor(endpoint: string, repositoryName: string, credential: TokenCredential, options?: ContainerRegistryClientOptions);
+    // (undocumented)
```

Comment:
It would be great if we could have the ref docs for the public api surface

Reply:
working on it!

Reply:
Think that will have done it, let me know if there are any other docs for the API surface that are missing.